README.md

```diff
@@ -1 +1,20 @@
 # hw01
+
+Hello, World! My name is Kwabene Kalumbula.
+
+For this, my first assignment in MACS 30500: Computing for the Social Sciences, I have been instructed to provide a short biograpghy. Thus, as the leader of a band of cartoon scallions once said: "Allow [me] to introduce [myself]." ![image of scallion](https://i.kym-cdn.com/entries/icons/original/000/027/174/veggie.jpg)
+
+#### Education
+
+* I was homeschooled K-12 (basically, this means that I was familiar with the above quote long before it became a meme)
+* I am now a third-year undergraduate at UChicago majoring in Sociology
+* While I have had to perform simple tasks in RStudio before, my computing experience is *very* limited. ![is this computer science?](https://i.guim.co.uk/img/media/44856f0611a6ae5e8f4e1875772d8f462a7e6f10/0_139_1118_671/master/1118.jpg?width=620&quality=85&auto=format&fit=max&s=7b41ac986fb0e6e8a8674946a915263d)
+
+## "Is this computer science?"
```

Comment:
Since you divide the rest of your bibliography into sections based on different parts of your background, using a header here makes it a bit confusing. While the comment is great, I think it would flow better if you perhaps italicized the text instead of putting it as a header.

Reply:
Agreed